data_model/clusters/TargetNavigator.xml

```diff
@@ -74,6 +74,7 @@ Davis, CA 95616, USA
     </enum>
     <struct name="TargetInfoStruct">
       <field id="0" name="Identifier" type="uint8">
+        <constraint type="maxLength" value="254"/>
```

Comment:
this seems off - why is there a max length constraint on a single integer value?

Reply:
looks like a typo. spec show the constraint as "all" values

Reply:
Removed.